src/axom/lumberjack/Message.hpp

```diff
@@ -22,13 +22,6 @@ namespace axom
 {
 namespace lumberjack
 {
-/*!
- *****************************************************************************
- * \brief Message to indicate no messages need to be sent from child node.
- *****************************************************************************
- */
-const char* const zeroMessage = "0";
```

Comment:
If I'm understanding the issue properly, would another possible fix be to keep the global variable `zeroMessage` as is, but copy it to a local variable before each usage?

(Not necessarily requesting any changes, just trying to understand the issue).

Reply:
Would it make sense to still have this pointer but point it to the storage that was added in Message.cpp with the same name? That way there is only one place that this is stored.

Reply:
I tried the following:
```
// Message.hpp
extern const char * zeroMessage;

---------------------------------------------------------
// Message.cpp
const char * zeroMessage = "0";
```
The same bug appears in this case. (Code-wise, I'm not sure if this is what you had in mind, if so, let me know!)

Reply:
You could also use a `std::array<char, 2> zeroMessage{"0"};` instead of a std::string to make it easy to make a local copy. Then use `zeroMessage.data()` when you need the pointer.

Reply:
> I was thinking something more like this.

Thanks @MrBurmark, that works and is pretty clean!